php/class-terminus-command.php

```diff
@@ -153,9 +153,14 @@ public static function request($realm, $uuid, $path = FALSE, $method = 'GET', $o
       );
       $cache->put_data($cachekey, $data);
       return $data;
-    } catch( Exception $e ) {
+    } catch( Guzzle\Http\Exception\BadResponseException $e ) {
       $response = $e->getResponse();
       \Terminus::error("%s", $response->getBody(TRUE) );
+    } catch( Guzzle\Http\Exception\BadResponseException $e ) {
+      $request = $e->getRequest();
+      \Terminus::error("Request %s had failed: %s", (string)$request, $e->getMessage() );
```

Comment:
I think you meant to delete lines 156-158 right? Also the reason I was using ```$response->getBody()```  is Guzzle throws the exception even when the api returns a valid error response and in those cases we want to know what the API said, not just what the Guzzle exception is. In fact, in most cases the Guzzle exception is useless. So I'd say check for response object first and if so return the body, otherwise return the guzzle exception.